java_gen/pre-written/pom.xml

```diff
@@ -131,20 +131,7 @@
                     </execution>
                 </executions>
             </plugin>
-            <!-- attach javadoc -->
```

Comment:
@rizard when I get the code from github and run the commands for build jar file, javadoc causes problems. I have to remove it. If it is mandatory, I can add and test the commands again.